packages/mongodb-memory-server-core/src/util/__tests__/MongoInstance.test.ts

```diff
@@ -209,13 +210,17 @@ describe('MongodbInstance', () => {
 
   describe('should work with mongodb LTS releases', () => {
     it('should work with mongodb 4.0', async () => {
-      const gotPort = await getPort({ port: 27445 });
-      const mongod = await MongodbInstance.create({
-        instance: { port: gotPort, dbPath: tmpDir },
-        binary: { version: '4.0.28' }, // explicit version instead of default to not mess it up later
-      });
-      expect(mongod.mongodProcess!.pid).toBeGreaterThan(0);
-      await mongod.stop();
+      const os = await getOS();
+
+      if (!isDebian10OrHigher(os)) {
+        const gotPort = await getPort({ port: 27445 });
+        const mongod = await MongodbInstance.create({
+          instance: { port: gotPort, dbPath: tmpDir },
+          binary: { version: '4.0.28' }, // explicit version instead of default to not mess it up later
+        });
+        expect(mongod.mongodProcess!.pid).toBeGreaterThan(0);
+        await mongod.stop();
+      }
```

Comment:
this is a change that should probably more properly be handled and across all those tests for platforms where it is not available

Reply:
Removed. Will do this in a separate PR.

Reply:
Actually I prefer to leave this to you. I don't have other platforms to test on.